Sources/MapboxCoreNavigation/RouteController.swift

```diff
@@ -254,13 +254,25 @@ open class RouteController: NSObject {
         rawLocation = location
         
         locations.forEach { navigator.updateLocation(for: FixLocation($0)) }
+    }
+    
+    @objc private func handleNavigationStatusNotification(_ notification: NSNotification) {
```

Comment:
Nit: Avoid “handle” in method names if possible; it’s a sign that the method could be named more descriptively or decomposed further. A common pattern for notification responder methods is e.g. `navigationStatusDidChange(_:)`.